BLSQ/iaso
# IA-3347: Change requests UX

![image](https://github.com/user-attachments/assets/4b46aa6f-4b17-4b3d-9da1-c8dc33aff365)


Related JIRA tickets : IA-3347

## Self proofreading checklist



## Changes

Moved content from the dialog to a dedicated page.

Adapt navigation to open a page instead of a modal

## How to test

Try to edit or see an existing change request.
If you can link a submission made with this form to test the rendering:
[formulaire_centre_de_sante_2024040401.xlsx](https://github.com/user-attachments/files/16944484/formulaire_centre_de_sante_2024040401.xlsx)


## Print screen / video


https://github.com/user-attachments/assets/dd34a9ad-031a-435a-9ba1-f91ad55764ee
